### PR TITLE
test(atomic): searchbox regression E2E tests

### PR DESCRIPTION
**Pull request by y-lakhdar**
To avoid re-introducing the bug fixed by https://github.com/coveo/ui-kit/pull/3582.

This is an issue that was brought up by more than one client, so let's not bring it back :P 

_Also. it is expected that the playwright E2E test is failing_

### CODE REVIEW

File: packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/page-object.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class AtomicCommerceSearchBoxLocators {
   }: {index?: number; total?: number; listSide?: 'Left' | 'Right'} = {}) {
     return this.page.getByLabel(
       new RegExp(
-        `suggested query\\. ${index ?? '\\d'} of ${total ?? '\\d'}\\.${this.listSideAffix(listSide)}`
+        `suggested query\\..*? ${index ?? '\\d'} of ${total ?? '\\d'}\\.${this.listSideAffix(listSide)}`
```
[the remark this answers is not in the document]
Comment (louis-bompart) [reply packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/page-object.ts:24]: Mh, odd, I'd like to know where the button come from.
But yeah we can tweak them!

[the remark this answers is not in the document]
Comment (fpbrault) [reply packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/page-object.ts:24]: I've fixed the selector in KIT-3246

[the remark this answers is not in the document]
Comment (fpbrault) [reply packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/page-object.ts:24]: The 'Button. ' comes from here :)
https://github.com/coveo/ui-kit/blob/2bd337b7c3890e387859471c095745a39e2b55f4/packages/atomic/src/components/search/atomic-search-box/search-suggestion.tsx#L90-L96

Comment (fbeaudoincoveo) [inline packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/page-object.ts:24]:
```suggestion
        `suggested query\\.( Button\\.)? ${index ?? '\\d'} of ${total ?? '\\d'}\\.${this.listSideAffix(listSide)}`
```

**Review by fbeaudoincoveo — APPROVED** [review]

**Pull request closed without merging**